lib/matplotlib/axis.py

```diff
@@ -1772,27 +1778,9 @@ def _set_ticklabels(self, labels, fontdict=None, minor=False, **kwargs):
             kwargs.update(fontdict)
         return self.set_ticklabels(labels, minor=minor, **kwargs)
 
-    def set_ticks(self, ticks, *, minor=False):
-        """
-        Set this Axis' tick locations.
-
-        If necessary, the view limits of the Axis are expanded so that all
-        given ticks are visible.
+    def _set_tick_locations(self, ticks, *, minor=True):
```

Comment:
It seems that `minor` is always passed, but why do you set a different default?

Reply:
Um, that's an error.

Changed the default to False. While it's always passed into the internal function, I still believe it makes sense to keep the signature parallel.